workspace/libs/shared/ui/src/lib/components/layout-header-footer/layout-header-footer.component.ts

```diff
@@ -8,8 +8,8 @@ import { PidpViewport, ViewportService } from '../../services';
   styleUrls: ['./layout-header-footer.component.scss'],
 })
 export class LayoutHeaderFooterComponent {
+  @Input() public providerIdentitySupport!: string;
```

Comment:
```suggestion
  @Input() public emailSupport!: string;
```
As this component is a library component, I think this property name should be more generic.
Somethings like `emailSupport`